.changeset/chatty-snails-begin.md

```diff
@@ -0,0 +1,5 @@
+---
+"@yamada-ui/avatar": minor
+---
+
+add role attribute for AvatarGroup
```

Comment:
I think chageset needs to start with a capital letter

Reply:
@cidkumagai 
thanks, fixed.